sn_node/src/node/messaging/proposal.rs

```diff
@@ -173,6 +181,40 @@ impl MyNode {
                     }
                 }
             }
+            Ok(Either::Right((serialised_proposal_1, serialised_proposal_2))) => {
+                let sig_share2 = if let Some(sig) = optional_sig_share {
+                    sig
+                } else {
+                    error!("Second sig share is missing! {}, {:?}", sender, msg_id,);
+                    return Ok(cmds); // TODO: return error here
+                };
+
+                let res1 = self
+                    .proposal_aggregator
+                    .try_aggregate(&serialised_proposal_1, sig_share);
+                let res2 = self
+                    .proposal_aggregator
+                    .try_aggregate(&serialised_proposal_2, sig_share2);
+                let res = (res1, res2);
+
+                match res {
```

Comment:
Unnecessary variable

```suggestion

                match (res1, res2) {
```

Reply:
Yeah I tested that first but got some compile error so just didn't spend any more time on it.
But I changed now and there was maybe something else causing it initially, because works fine.